### PR TITLE
add instruction to choose implicit checkbox

### PR DESCRIPTION
**Pull request by mystiberry-okta**
Fixes OKTA-96836.

### CODE REVIEW

**Comment by mystiberry-okta** [comment]
@larsjohansen-okta , please review for fix to doc bug https://oktainc.atlassian.net/browse/OKTA-96836.

**Comment by mystiberry-okta** [comment]
@larsjohansen-okta Ping. If you are swamped, can someone else review?

File: _source/docs/api/resources/social_authentication.md [hunk 1]
```diff
@@ -88,7 +88,8 @@ Advantages of using Okta Social Authentication include:
     3. Select *Add Application*.
     
     4. Select *Create New App* to launch the App Integration Wizard. It guides you through the necessary configuration
-    steps and give you back a `client_id` which you use in Step 7.
+    steps and give you back a `client_id` which you use in Step 7. Make sure that in the **General Settings** tab, you
+    check the **Implicit** checkbox for allowed grant types.
```
Comment (larsjohansen-okta) [inline _source/docs/api/resources/social_authentication.md]: I think this is good info, however I think we should also add a little note that you need to select the OpenID Connect app type once you get the modal open from clicking on "Create New App" button. See screenshot: 
image

**Comment by mystiberry-okta** [comment]
@larsjohansen-okta is this okay? I didn't put the screenshot in the doc because then it seemed like I should add a bunch more screenshots and wasn't sure that'd be helpful (the wizard is super easy to understand).

**Comment by larsjohansen-okta** [comment]
If the layout is fine, I think this looks good. 🚢

**Comment by mystiberry-okta** [comment]
Layout with numbers looks funky, but the HTML looked right when I tested it. Thanks!

**Pull request merged**